.github/workflows/cleandocker.yml

```diff
@@ -0,0 +1,29 @@
+name: CleanDocker
```

Comment:
Nice initiative. We can create a reusable action for this for all our repos with similar behaviour 😄 💯

Reply:
Would recommend opting for a Lifecycle policy in the Registry , instead of handling this as part of CI.

Reply:
Who's team would be in charge of setting up the lifecycle, so I can ping them here?

Reply:
Feel free to raise a ticket under this board. 
https://mattermost.atlassian.net/jira/software/c/projects/CLD/boards/112?assignee=62472e8d258562006fa4e1c8

Describing the lifecycle policy required for retaining artifacts under `mattermost/bifrost` repo.

Reply:
cc: @gabrieljackson can you take a hold on this? Since it should be migrated to all other cloud repos as well.